elt-pipelines/fase/ingest/fase/utils/oracle.py

```
@@ -0,0 +1,160 @@
+                pa_type = pa.timestamp("us")
+            elif pq_type_str == "date":
+                pa_type = pa.date32()
+            else:
+                pa_type = pa.string()
+            arrow_fields.append(pa.field(col_name.lower(), pa_type, nullable=True))
+
+        target_schema = pa.schema(arrow_fields)
+
+        with self.engine.connect() as conn:
+            query = (
+                f'SELECT * FROM "{self.config.schema.upper()}"."{table_name.upper()}"'
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)